src/app/shared/detail-data/detail-data.component.html

```diff
@@ -0,0 +1,7 @@
+<article>
+  <section class="article-container" *ngIf="detailData">
+    <img class="article-profile-image" src="{{detailData?.thumbnail?.path}}.{{detailData?.thumbnail?.extension}}"
```

Comment:
can you construct the url and then send it? so the src can be "imgUrl"

Reply:
Okay